README

```diff
@@ -128,14 +128,41 @@ users.
 PYTHON INSTALL
 --------------
 The GEOPM python tools are packaged in the RPMs described above, but
-they are also available from PyPI as the `geopmpy` package.  For
-example, to install the geopmpy package into your home directory, run
-the following command:
+the latest release is also available from PyPI as the `geopmpy`
+package.  For example, to install the geopmpy package into your home
+directory, run the following command:
 
-    pip install --user geopmpy
+    python -m pip install --user geopmpy
 
-Note this installs only the GEOPM python tools and does not install
-the full GEOPM runtime.
+To install the latest development version of the `geopmpy` package
```

Comment:
Explain that 2.0 release is coming soon, and that many of the 2.0 features are not available from PIP.